Gemfile

```diff
@@ -63,7 +63,7 @@ gem 'nokogiri-diff', '~> 0.2.0' # for comparing xml documents
 
 gem 'activerecord-import' # bulk insertion of data
 
-gem 'activerecord-session_store'
+gem "activerecord-session_store", ">= 2.0.0"
```

Comment:
Do we have to specify the version here? If we just regenerate the lock file from scratch, wouldn't it pick up the latest 2.0.0 version?